### PR TITLE
sync-diff-inspector: Clear some codes and optimize GetRandomValues

### PR DESCRIPTION
**Pull request by Leavrth**
### What problem does this PR solve? 
Issue Number: ref #670 

1. There are some fields and functions not used.
2. dbutil.GetRandomValues only get a column each time, we can get the entire rows instead.

### What is changed and how it works?

1. remove them.
2. optimize.

### CODE REVIEW

File: sync_diff_inspector/config/config.go [hunk 1]
```diff
@@ -180,12 +180,8 @@ func (t *TaskConfig) Init(
 		return errors.Trace(err)
 	}
 
-	// Set default value when output is empty
 	if t.OutputDir == "" {
-		t.OutputDir = timestampOutputDir()
-		if err := os.RemoveAll(t.OutputDir); err != nil && !os.IsNotExist(err) {
-			log.Fatal("fail to remove the temp directory", zap.String("path", t.OutputDir), zap.String("error", err.Error()))
-		}
+		return errors.New("output-dir is missing from the task configuration")
```
Comment (3pointer) [inline sync_diff_inspector/config/config.go]: 🤔 seems we will generate a temporary output-dir

Comment (Leavrth) [reply sync_diff_inspector/config/config.go]: PTAL again

**Review by 3pointer — APPROVED** [review]
LGTM

**Review by lichunzhu — APPROVED** [review]
LGTM

**Pull request merged**